lib/version-map.json

```diff
@@ -0,0 +1,16 @@
+{
+	"plugins": {
+		"ejs": ">= 2.0.0",
+		"mustache": "1.1.0 - 2.0.0",
+		"stache": ">= 2.1.0"
+	},
+	"jquery": {
+		"1.7.1": "1.0.0",
```

Comment:
Do we need this mapping? Shouldn't all 2.x work with jQuery 2 (or 1.11) and all 1.x with jQuery 1.x? I don't think the compilation uses anything specific between those versions.